enterprise/backend/test/metabase_enterprise/metabot_v3/api_test.clj

```diff
@@ -5,10 +5,288 @@
    [metabase-enterprise.metabot-v3.client :as metabot-v3.client]
    [metabase-enterprise.metabot-v3.config :as metabot-v3.config]
    [metabase-enterprise.metabot-v3.tools.api :as metabot-v3.tools.api]
-   [metabase.test :as mt]))
+   [metabase.test :as mt]
+   [toucan2.core :as t2]))
+
+(defmacro with-clean-metabots
```

Comment:
I think you can use `(mt/with-model-cleanup [:model/Metabot] ...)` and it should do the same thing you're doing here

Reply:
Actually this macro does something else. I want to remove the metabot models for this test, so I'm not worried about cleaning up anything just deleting those pre-existing models and restoring them after this test.